src/components/ThumbsUp.jsx

```diff
@@ -1,5 +1,21 @@
-const ThumbsUp = () => {
-  return <button>Thumbs Up</button>;
+import { patchVotesInc } from "../utils/API";
+
+const ThumbsUp = ({setSingleReview, singleReview, review_id}) => {
+
+const thumbsUpHandler = () => {
+
+  setSingleReview(currReview => {
+    return {...currReview, votes: currReview.votes + 1 }
+  })
+
+  patchVotesInc(review_id).then((updatedReview) => {
+  }).catch(err => {
```

Comment:
Consider adding a message to the user to alert them that the request has failed.